src/dmd/dmodule.d

```diff
@@ -579,6 +579,16 @@ extern (C++) final class Module : Package
         else
         {
             const(char)* argdoc;
+            OutBuffer buf;
```

Comment:
This (and the string inside) is scoped, but the string should always be copied (by `forceExt`) before the scope exits.